### DISCUSSION TITLE
images not displaying in the browser

### DISCUSSION THREAD

**Discussion by pascaloseko** [minio/minio#18023] [Q&A]
I have minio container running in my virtual machine, and the images uploaded were viewable in the client app but at some point, they weren't showing.

I restarted the container but the issue was not resolved. The URL `http://185.209.228.41:9000/rvtn/2/182/Screenshot_20230816-131516_Jumia.jpg`  is not accessible 

can someone please help me out with this?

**Comment by ravindk89** [comment]
You need to provide a lot more information here. The version of MinIO you are running, how you are uploading files, how you are generating that URL (e.g. presigned or via making the bucket public), and how the client access those files.

From there someone can try to give more guidance. as an aside, please do not at-mention users who are not actively involved in the current discussion. All support is best-effort only.
